### DISCUSSION TITLE
Log-based Alerts, e.g. for slow queries

### DISCUSSION THREAD

**Discussion by psteinroe** [supabase/supabase#9231] [Feature Requests]
Hey,

While the range of possibilities offered by a sql-based log analyzer is huge, I'd like to see a way to automatically alert on certain conditions. Right now we check the logs manually, but it would be great if we had notifications for basic use cases, like slow queries.

**Comment by thorwebdev** [comment]
This is a great idea, thanks! I've flagged with the team and will have them follow up.

**Comment by Ashwin-Khowala** [comment]
https://jsr.io/@ashwinkh/supabase-alerts-cli
was facing something similar, so I made this.
